### DISCUSSION TITLE
Option per keybinding override to remove the state popup

### DISCUSSION THREAD

**Discussion by ksandvik** [helix-editor/helix#4455] [General]
Is there a way to remove the display of the popup per key each mapping bindings, let's say go mode gl is mapped to something else, and there's no need to show the flash of the go state keybinding popup?

**Comment by CptPotato** [comment]
This isn't quite what you're asking for, but there's a PR for adding a delay to the command popup: #2715

**Comment by ksandvik** [comment]
Maybe if there's on option of delay 0 ms, then it will not show up at all. I will update the PR, thx. Even better if it's per command binding.
